File: Example/Sources/OtherClass.cs
using System;
using System.Collections.Generic;

namespace DataTransferObjects
{
  public class OtherClass
  {
    public class InnerClass
    {
      public static string StaticField = "789";

      public static string StaticMethod()
      {
        return "aaa";
      }

      public static string StaticMethodArg(string name)
      {
        return "hellow!" + name;
      }
    }
    public static string StaticField = "456";
    public static string StaticMethod()
    {
      return "bbb";
    }
  }
}

File: Example/ConvertResults/All/OtherClass.ts
export class OtherClass {
  public static StaticField: string = "456";
  public static StaticMethod(): string {
    return "bbb";
  }
}
export namespace OtherClass {
  export class InnerClass {
    public static StaticField: string = "789";
    public static StaticMethod(): string {
      return "aaa";
    }
    public static StaticMethodArg(name: string): string {
      return "hellow!" + name;
    }
  }
}
